docs/guides/UpgardeGuide-v5/UpgradePart2.md

```diff
@@ -0,0 +1,1012 @@
+      /check-spec
++       /input[default].png     <-- added
++       /input[with_input].png  <-- added
+        /table_default[default].png
++       /table_with_first_item_selected[row_1_selected].png <-- added
+        /table_with_second_item_selected[row_2_selected].png
+        /popup[default].png
+-       /click_popup_content[default].png  <-- removed
++       /popup[click_popup_content].png    <-- added
+    /chrome_huge
+      /check-spec
++       /input[default].png     <-- updated to be terra huge size
++       /input[with_input].png  <-- updated to be terra huge size
+        /table_default[default].png
+        /table_with_first_item_selected[row_1_selected].png
+        /table_with_second_item_selected[row_2_selected].png
+        /popup[default].png
+-       /click_popup_content[default].png  <-- removed
++       /popup[click_popup_content].png    <-- added
+    /firefox_tiny  <-- same as chrome_tiny
+    /firefox_huge  <-- same as chrome_huge
+```
+</details>
+
+## Test Update Gotchas
+So these test updates are all hunky dory, however there are a few things that might confuse you in your screenshots. 
+
+### 1. Orange Focus Glow?
+
+This is the browser focus glow color on native html element defined by Google Chrome.
+
+###  2. Unexpected hover styles
+
+Since there is no global refresh, the mouse hover may be present. This is fine, you can take screenshot with hover styles or you can move the mouse with the wdio [browser.moveToObject() command](http://v4.webdriver.io/api/action/moveToObject.html). Example:
+
+```
+browser.moveToObject('#root', 0, 400); // move mouse to (0,400) offset of the #root element
+```
+
+###  3. Screenshot Show Visuals from The Previous Test
+
+Since there is no global refresh, the state on the page is continuous. This means previous test interactions will effect the next test that uses the same test url.
+
+This will allow for better test coverage and enables one to slim down and write more coherent tests (we reduced 150 lines of code for terra-popup's spec).
+
+You can reorganize and/or rewrite tests to capture the UI behavior expected in the screenshots.
+
+###  4. Errors When Clicking a Table Row in Firefox
+
+Unfortunately clicking <tr> tags is a known Firefox bug. Firefox maintainers suggested work around: Using the center <td> cell within <tr> as the click target which would be equivalent to clicking the center of the <tr> cell directly. See [geckodriver issue](https://github.com/mozilla/geckodriver/issues/1228) and/or [firefox issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1448825) for more info.
+
+Our suggested workaround this issue by using the [`browser.leftClick()` command](http://v4.webdriver.io/api/action/leftClick.html) or using keyboard navigation to interact with a table row.
+
+The wdio `browser.leftClick()` works because its using used button press and the W3C actions protocol API behind the scenes where as of the `browser.click()` command used the click protocol API directly.
```

Comment:
```suggestion
The wdio `browser.leftClick()` works because its using the button press and the W3C actions protocol API behind the scenes where as of the `browser.click()` command used the click protocol API directly.
```